.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": [
+      "react-app",
+      "airbnb",
+      "plugin:jsx-a11y/recommended",
+      "plugin:prettier/recommended"
+    ],
+    "plugins": [
```

Comment:
move all of this into a folder named frontend.
We are going to have a backend and a frontend folder in root.

Reply:
sure noted